include/ensmallen.hpp

```diff
@@ -34,6 +34,10 @@
 
 #include <armadillo>
 
+#ifdef USE_COOT
```

Comment:
@zoq Suggest changing `USE_COOT` to `ENS_USE_COOT` here and in CMakeLists.txt.

This is for consistency with all the other macros used by ensmallen, where the `ENS_` prefix is used.  The prefix helps avoid potential clashes.

Reply:
Makes sense, will change it.